java/de/yessoft/android/mohammad/application/app.java

```diff
@@ -0,0 +1,58 @@
+package de.yessoft.android.mohammad.application;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.FirebaseFirestoreSettings;
+
+public class app extends Application {
```

Comment:
you should use dependent injection , you may research which di is mostly used in android word

Reply:
OK, i would do that when i learn dagger :) however i'm working on it